frontend/src/components/filters/descriptions.js

```diff
@@ -648,7 +648,7 @@ export const POOLED_SAMPLES_FILTERS = {
     },
     project_name: {
         type: FILTER_TYPE.INPUT,
-        key: "derived_sample__project__name",
+        key: "project__name",
```

Comment:
I was wondering if this filtering made sense. Turns out, pooled-sample works with DerivedBySample so it's good.